Weekly Update 3.md

```diff
@@ -0,0 +1,49 @@
+# Update for the 3rd week of self reports
+
+## 1.
+- This week I mostly accomplished this weeks goals
+- I successfully imported my two made files into their own dataframes
+- I was able to join them based on their like words, and export that dataframe to a new excel sheet.
+- I coded my scraper and my other program to remove stop words, which will get it ready for NLTk sentiment analysis.
+
+## 2. 
+
+- It takes a long time to run the programs
+
+- I wasn't intending to remove stop words yet until I saw the merged data and realized some words showed up that really didn't need to.
+
+- I'm considering taking data from other sources, but also realize that because of the timeconstraint it may be better to continue for now and improve on technique later
+
+
+## 3. 
+
+- I want to integrade sentiment analysis into my webscraping code   -2 days
```

Comment:
You might wanna look into NLTK pre-trained sentiment analyzer. It's a built-in, pretrained sentiment analyzer called VADER (Valence Aware Dictionary and sEntiment Reasoner)

Here's an article on it: https://realpython.com/python-nltk-sentiment-analysis/